Automattic/wp-calypso
# Full Site Editing: prevent access to template CPTs

#### Changes proposed in this Pull Request

**Note:** not to be merged until we have a way to automatically fill in the required template data (https://github.com/Automattic/wp-calypso/pull/33878)

For initial milestones we don't want to expose template and template part
options in the sidebar.

#### Testing instructions

* Activate the Full Site Editing plugin on your test site.
* Make sure that Template and Template Part items are not present in the wp-admin sidebar anymore.
* Navigate to edit page view and verify that template selector sidebar is no longer visible.

Fixes https://github.com/Automattic/wp-calypso/issues/33633
